src/pages/AppContainer.tsx

```diff
@@ -140,7 +140,7 @@ const AppContainer: React.FunctionComponent = () => {
     []
   );
 
-  const onChangeSoundConfig = (event: React.ChangeEvent<HTMLInputElement>) => {
+  const onChangeSoundConfig = () => {
```

Comment:
no-unused-vars

Reply:
Personally I feel the argument type is meaningful information.
So how about to chnage as below?

```suggestion
  const onChangeSoundConfig = (_event: React.ChangeEvent<HTMLInputElement>)
```

Reply:
I also think the argument type is meaningful but the change you suggested still warned by the eslint.
Is there any option to suppress such unused vars by heading underscore? I'll check again.

Reply:
Did it! Addressed in https://github.com/mobu-of-the-world/mobu/pull/82/commits/355aa63473bdd953aaa7adfb8628410cba8b17e9

Reply:
Great!